src/App.js

```diff
@@ -14,11 +14,110 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    goods: goodsFromServer,
+    startButton: false,
```

Comment:
It's a bad name and it's hard to understand for what purposes is used.
This variable actually represents if the list is visible or not, so it would be better to call it smth like 'isListVisible'